data/migrations/20211026100908_profiles.js

```diff
@@ -1,28 +1,10 @@
 exports.up = (knex) => {
-  return knex.schema
-    .raw('CREATE EXTENSION IF NOT EXISTS "uuid-ossp"')
-    .createTable('profiles', function (table) {
-      table.string('profile_id').notNullable().unique().primary();
-      table.string('email').notNullable().unique();
-      table.string('first_name');
-      table.string('last_name');
-      table.string('location');
-      table.string('company');
-      table.specificType('tech_stack', 'text ARRAY');
-      table
-        .integer('role_id')
-        .unsigned()
-        .notNullable()
-        .default(5)
-        .references('role_id')
-        .inTable('roles')
-        .onDelete('CASCADE')
-        .onUpdate('CASCADE');
-      table.timestamps(true, true);
-      table.boolean('is_active');
-      table.string('progress_status');
-      table.float('attendance_rate').notNullable().unsigned().default(1.0);
-    });
+  return knex.schema.createTable('profiles', function (table) {
+    table.string('user_id').notNullable().unique().primary();
+    table.string('profile_id').notNullable().unique();
+    table.string('role').notNullable();
+    table.timestamps(true, true);
+  });
```

Comment:
Nice cleanup

Reply:
I agree with @roxannebobby this simplifies the profs schema a ton. I also like that role is now a part of the schema on profiles. One table, all user info that is required beyond what we get from the DS API.